05-ethics.Rmd

```diff
@@ -5,12 +5,12 @@ output:
 ---
 # Ethics
 
-Ethics refers to a set of moral principles that dictate a person's behavior. While the field of ethics is often considered to be a theoretical discipline, ethical conduct is an important objective in practice as well. In the field of data visualization, there are many opportunities to manipulate viewers with untruthful representations of data; thus, like any other discipline, data visualization faces significant ethical challenges, of which we will explore many in this chapter.
+This chapter covers the ethical implication to data visualizations. Ethics refers to a set of moral principles that dictate a person’s behavior. While the field of ethics is often considered to be a theoretical discipline, ethical conduct is an important objective in practice. In the field of data visualization, there are many opportunities to manipulate viewers with untruthful representations of data; thus, like any other discipline, data visualization faces significant ethical challenges. This chapter will touch on the importance of ethics in visualization, guidelines for ethical visualization, topics relating to data deception, ethical challenges faced when creating visualization, visualization and social change, and more.  
```

Comment:
I prefer your update over the original since it gives an overview of what we'll be covering in the chapter. However, I think it'd be better if we list out the topics as bullets.

Reply:
@syragrg is this format of listing out topics consistent with other chapters? If yes, I'll go ahead and approve the changes. LMK!

Reply:
The other chapters starts with "This chapter ......" so I changed that part. However, the other chapter's intro is more brief it actually doesn't list the topics (headers essentially) that it will be covering..

Reply:
Just had a quick look at other chapters; I quite like this format and it looks consistent, so I'll approve it!